docs/index.md

```diff
@@ -4,6 +4,20 @@ layout: default
 sort: 1
 ---
 
-# Introduction
+# Node Feature Discovery operator
 
-***COMING SOON.***
+The Node Feature Discovery operator manages detection
+of hardware features and configuration in a kubernetes cluster.
+The operator orchestrates all resources needed to run the
+[Node-Feature-Discovery](https://github.com/kubernetes-sigs/node-feature-discovery)
+DaemonSet
+
+Continue to:
+
+- **[Quick start](get-started/index.md)** for quick step-by-step
+  instructions on how to get the NFD-Operator running on your cluster.
+
+- **[Advanced](advanced/index.md)** for Advanced topics and reference.
+
+- **[Contributing](contributing/index.md)** for information on how to reach the
+  community, project Governance and License info.
```

Comment:
These links lead to empty-ish pages with no links or any "path forward". If we want to keep this index page the sub-index pages (get-started/index.md etc) should contain a toc, I think.

Reply:
I agree, will check how to do a toc in Jekyll ;P